ports/stm32/boards/NUCLEO_H563ZI/mpconfigboard.h

```diff
@@ -0,0 +1,80 @@
+#define MICROPY_HW_BOARD_NAME               "NUCLEO_H563ZI"
+#define MICROPY_HW_MCU_NAME                 "STM32H563ZI"
+
+#define MICROPY_PY_PYB_LEGACY               (0)
+#define MICROPY_HW_ENABLE_INTERNAL_FLASH_STORAGE (1)
+#define MICROPY_HW_ENABLE_RTC               (1)
+#define MICROPY_HW_ENABLE_RNG               (1)
+#define MICROPY_HW_ENABLE_ADC               (1)
+#define MICROPY_HW_ENABLE_DAC               (0) // TODO: requires DMA, which is not yet ready
+#define MICROPY_HW_ENABLE_USB               (1)
+#define MICROPY_HW_HAS_SWITCH               (1)
+#define MICROPY_HW_HAS_FLASH                (1)
+
+// The board has a 8MHz oscillator, the following gives 250MHz CPU speed
+#define MICROPY_HW_CLK_USE_BYPASS           (1)
+#define MICROPY_HW_CLK_PLLM                 (2)
+#define MICROPY_HW_CLK_PLLN                 (125)
+#define MICROPY_HW_CLK_PLLP                 (2)
+#define MICROPY_HW_CLK_PLLQ                 (2)
+#define MICROPY_HW_CLK_PLLR                 (2)
+#define MICROPY_HW_CLK_PLLVCI_LL            (LL_RCC_PLLINPUTRANGE_4_8)
+#define MICROPY_HW_CLK_PLLVCO_LL            (LL_RCC_PLLVCORANGE_WIDE)
+#define MICROPY_HW_CLK_PLLFRAC              (0)
+
+// PLL3 with Q output at 48MHz for USB
+#define MICROPY_HW_CLK_PLL3M                (25)
+#define MICROPY_HW_CLK_PLL3N                (192)
+#define MICROPY_HW_CLK_PLL3P                (2)
+#define MICROPY_HW_CLK_PLL3Q                (4)
+#define MICROPY_HW_CLK_PLL3R                (2)
+#define MICROPY_HW_CLK_PLL3FRAC             (0)
+#define MICROPY_HW_CLK_PLL3VCI_LL           (LL_RCC_PLLINPUTRANGE_1_2)
+#define MICROPY_HW_CLK_PLL3VCO_LL           (LL_RCC_PLLVCORANGE_MEDIUM)
+
+// 5 wait states, according to Table 37, Reference Manual (RM0481 Rev 1)
+#define MICROPY_HW_FLASH_LATENCY            FLASH_LATENCY_5
+
+// There is an external 32kHz oscillator
+#define MICROPY_HW_RTC_USE_LSE              (1)
+
+// UART config
+// TODO: UART1 or LPUART1? LPUART1 seems to crash system at boot...
```

Comment:
Using UART1 is fine.

Please then remove this TODO comment.

Reply:
Done